backend/src/main/java/com/htecgroup/skynest/exception/UserExceptionType.java

```diff
@@ -13,6 +13,8 @@ public enum UserExceptionType {
   INVALID_EMAIL_OR_PASSWORD("The email and password are not valid", HttpStatus.BAD_REQUEST),
   PASSWORDS_DOES_NOT_MATCH(
       "The provided password doesn't match the username", HttpStatus.EXPECTATION_FAILED),
+  OLD_PASSWORD_IS_INCORRECT(
+      "The provided old password is incorrect", HttpStatus.EXPECTATION_FAILED),
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/417

we really need to use more meaningful error codes

leave it to @DraganMitrasinovicHtec

Reply:
Ma hajde promenicu to, iskopirao i zaboravio..

Reply:
It is better if you don't provide the info that the email is correct but the password is not. If you see all apps respond with **_invalid email or password_** when your credentials are wrong regardless which field is wrong.. But you need to consult with FE and QA team for that change